app/controllers/application_controller.rb

```diff
@@ -50,10 +50,13 @@ def current_session_id
   end
 
   def strip_empty_checkboxes(form_key, fields)
-    if params[form_key].present?
-      fields.each do |field|
-        params[form_key][field] = params[form_key][field]&.reject(&:blank?) unless params[form_key][field].is_a?(String)
-      end
+    params_to_strip = if form_key.present? && params[form_key].present?
```

Comment:
shouldn't checking that `params[form_key].present?` is present  to be enough or you need to check also for form_key.present?

Reply:
It was a bit lazy from me, let me do something better